include/OpenSpaceToolkit/Astrodynamics/Flight/System/SatelliteSystemBuilder.hpp

```diff
@@ -39,7 +39,15 @@ using ostk::astro::flight::System;
 using ostk::astro::flight::system::SatelliteSystem;
 using ostk::astro::flight::system::PropulsionSystem;
 
-/// @brief                      Satellite System builder
+/// @brief                      Satellite System builder, meant to simplify creation of a SatelliteSystem, by allowing 
+///                             you to only specify the parameters you want. There are two ways of doing this:
+///                             
+///                             Chaining together your desired parameters like so:
+///                             SatelliteSystemBuilder().withDryMass(X).withArea(Y)
```

Comment:
you can use the `@code` ... `@endcode` notation to document code examples:

![image](https://github.com/open-space-collective/open-space-toolkit-astrodynamics/assets/65550121/387d542d-c66f-4d67-8570-6883b18b0da5)